pkg/remoteclient/fake.go

```diff
@@ -25,11 +25,11 @@ type fakeBuilder struct {
 // Build returns a fake controller-runtime test client populated with the resources we expect to query for a
 // fake cluster.
 func (b *fakeBuilder) Build() (client.Client, error) {
-
-	scheme, err := buildScheme()
+	// FIXME Remove? Do we need buildScheme()?
```

Comment:
Hah, beautiful: `buildScheme()` is doing exactly what your new thing is made for, only for a subset of code paths. Yes, get rid of this code as well as `buildScheme()` itself.